### PR TITLE
Remove Android comments

### PR DESCRIPTION
**Pull request by xsacha**

### CODE REVIEW

**Comment by ruslo** [comment]
I forget to update pkg.openssl branch, sorry. Please rebase.

**Comment by xsacha** [comment]
Rebased
Success: https://travis-ci.org/ingenue/hunter/jobs/169384935

File: .travis.yml [hunk 1]
```diff
@@ -70,11 +67,8 @@ matrix:
       env: PROJECT_DIR=examples/OpenSSL TOOLCHAIN=osx-10-11
     - os: osx
       env: PROJECT_DIR=examples/OpenSSL TOOLCHAIN=ios-nocodesign-9-3
-
-    # FIXME:
-    # * https://travis-ci.org/ingenue/hunter/jobs/164816649
-    # - os: osx
-    #   env: PROJECT_DIR=examples/OpenSSL TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
+    - os: osx
+      env: PROJECT_DIR=examples/OpenSSL TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
```
Comment (ruslo) [inline .travis.yml]: Not working: https://travis-ci.org/ingenue/hunter/jobs/169384946

**Comment by xsacha** [comment]
Strange, it failed on Mac.
Edit: https://github.com/ruslo/hunter/issues/260

I see. It is a different configure script for Mac.

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by ruslo** [comment]
> Has the old error about HAVE_POLL_FINE_EXITCODE not being set. Code not updated?

As far as I remember that was about CURL, no?

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1108
- https://travis-ci.org/ingenue/hunter/builds/169449967

**Pull request merged**